clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -64,6 +64,8 @@
 
     private static final Logger log = LoggerFactory.getLogger(ConsumerCoordinator.class);
 
+    private static final long CLOSE_TIMEOUT_MS = 5000;
```

Comment:
I will raise a KIP to add a close with timeout for consistency with the producer. Can we do that in a follow-on JIRA since this is already quite a big change? The hard-coded limit can be temporary until a close() with timeout is implemented.